third_party/move/move-prover/tests/sources/functional/resource_index.exp

```diff
@@ -0,0 +1,6 @@
+Move prover returns: exiting with model building errors
+error: not supported before language version `2.0-unstable`: resource indexing
```

Comment:
Can you align the error message with those emitted by parser/expansion elsewhere? You have some tests which show how they look like in this PR. It mentions 'Move 2' instead of unstable. 

In general it appears though the prover tests should turn on language version 2 to get rid of those errors?